benchmarking/example_classification.py

```diff
@@ -10,33 +11,37 @@
 from sklearn.metrics import roc_auc_score
 from sklearn.preprocessing import MinMaxScaler
 
-from skfp import (
-    AtomPairFingerprint,
-    ECFPFingerprint,
-    ERGFingerprint,
-    MACCSKeysFingerprint,
-    MAP4Fingerprint,
-    MHFPFingerprint,
-    TopologicalTorsionFingerprint,
-)
-
-dataset_names = [
-    "ogbg-molhiv",
-    "ogbg-molbace",
-    "ogbg-molbbbp",
+import skfp.fingerprints
+
+fingerprint_classes = [
+    cls
+    for name, cls in inspect.getmembers(skfp.fingerprints, predicate=inspect.isclass)
 ]
-property_names = [
-    "HIV_active",
-    "Class",
-    "p_np",
+
+SIZE = 125
+
+dataset_params = [
+    ("ogbg-molhiv", "HIV_active"),
+    ("ogbg-molbace", "Class"),
+    ("ogbg-molbbbp", "p_np"),
 ]
 
-for dataset_name, property_name in zip(dataset_names, property_names):
+
+classifier_parameters = [
+    (RandomForestClassifier, {"n_jobs": -1, "class_weight": "balanced"}),
+    (
+        LogisticRegression,
+        {"class_weight": "balanced", "penalty": None, "n_jobs": -1},
```

Comment:
I would use `LogisticRegressionCV` here, with and default tuning of L2 penalty. Linear classifiers are sensitive to regularization strength

Reply:
We can change `LogisticRegression` to `LogisticRegressionCV`. About L2: we already use it - it's default.